### PR TITLE
feat: add more details about project generation

### LINKED ISSUE #3: The cli need to init an empty git repository, it will allow husky to install git hooks properly

### PR DESCRIPTION
**Pull request by DecampsRenan**
## Done
- Added prettier config from start-ui-web
- Setup eslint, husky and lint-staged
- Added StartUI logo as CLI header
- Rewrite help text
CleanShot 2021-09-29 at 16 41 15@2x

- Added details about what's done during generation
https://user-images.githubusercontent.com/7521065/135292211-7dbe233e-044b-486e-93b1-e25758debd41.mp4

Resolve #3

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+	"semi": true,
 	"singleQuote": true,
-	"printWidth": 80,
-	"semi": true
+	"tabWidth": 2,
+	"trailingComma": "es5",
+	"arrowParens": "always",
+	"importOrder": [
+		"^react$",
+		"^(?!^react$|^@/|^[./]).*",
```
Comment (yoannfleurydev) [inline .prettierrc.json]: If think this is useless as there is no React in this project.

File: package.json [hunk 2]
```diff
@@ -33,19 +35,26 @@
...
 	},
+	"lint-staged": {
+		"*.{js,json}": "prettier --write",
+		"*.{js}": "eslint --cache --fix"
+	},
 	"engines": {
 		"node": ">=14"
 	}
-}
+}
```
Comment (yoannfleurydev) [inline package.json]: No newline at end of file.

**Review by yoannfleurydev — APPROVED** [review]
There is also multiple file that are still using tabs, and I think that we are more for spaces, is it possible to prettier them so they use spaces instead ?

File: .prettierrc.json (hunk 1 above)
Comment (DecampsRenan) [reply .prettierrc.json]: Oh yes, I will remove the entire plugin because it doesn't seems to work with `require`

File: package.json (hunk 2 above)
Comment (DecampsRenan) [reply package.json]: Fixed with a prettier pass ;)

**Comment by yoannfleurydev** [comment]
bitmoji

**Pull request merged**